Packs/D2/pack_metadata.json

```diff
@@ -1,20 +1,19 @@
 {
-    "name": "D2",
-    "description": "D2 Agent pack",
-    "support": "xsoar",
-    "currentVersion": "1.0.0",
-    "author": "Cortex XSOAR",
-    "url": "https://www.paloaltonetworks.com/cortex\",",
-    "email": "",
-    "categories": [
-        "Utilities"
-    ],
-    "tags": [],
-    "created": "2020-05-14T14:23:06Z",
-    "updated": "2020-05-14T14:23:06Z",
-    "beta": false,
-    "deprecated": false,
-    "useCases": [],
-    "keywords": [],
-    "dependencies": {}
+  "name": "D2",
+  "description": "D2 Agent pack",
+  "support": "xsoar",
+  "currentVersion": "1.0.0",
```

Comment:
The indentation should be 4 spaces.
And why the version is not `1.0.1`?

Reply:
As long as we are moving items between packs, better to stay with initial version. It is causing issue in later step